docs/src/background.md

```diff
@@ -48,7 +48,7 @@ where ``\hat{\bm{g}} \in \mathbb{C}^\mathbf{N}``.
 
 ## NFFT
 
-The NFFT is an approximative algorithm that realizes the NDFT in just ``{\cal O}(N \log N + M)`` steps where ``N := \text{prod}(\bm{N})``. This is at the same level as the ordinary FFT with the exception that of the additional linear dependence on ``M`` which is unavoidable since all nodes need to be touched as least once.
+The NFFT is an approximative algorithm that realizes the NDFT in just ``{\mathcal O}(N \log N + M)`` steps where ``N := \text{prod}(\bm{N})``. This is at the same level as the ordinary FFT with the exception that of the additional linear dependence on ``M`` which is unavoidable since all nodes need to be touched as least once.
```

Comment:
```suggestion
The NFFT is an approximative algorithm that realizes the NDFT
in just ``{\mathcal O}(N \log N + M)`` steps 
where ``N := \text{prod}(\bm{N})``.
This is at the same level as the ordinary FFT
with the exception that of the additional linear dependence on ``M``
which is unavoidable since all nodes need to be touched as least once.
```

Reply:
Breaking it into phrases per line means the next time a change is needed the git diff will be more concise.
Just bike shedding :)
LGTM.

Reply:
I agree with you but that doesnt seem to match the style of the rest of the document, so maybe leave applying word-wrap (everywhere) for a potential separate commit? Anyway just going to let you decide, was just passing by and caught this the typo